wakuv2/waku.go

```diff
@@ -536,15 +535,15 @@ func (w *Waku) runPeerExchangeLoop() {
 	}
 }
 
-func (w *Waku) getPubsubTopic(topic string) string {
+func (w *Waku) GetPubsubTopic(topic string) string {
```

Comment:
Wondering if exposing this outside would cause issues? As this logic can change as sharding gets implemented.